spec/ttml1.xml

```diff
@@ -4303,6 +4303,12 @@ glyphs do not have an EM square that is distinct from their font's EM square.</p
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+<p><loc href="#style-value-length">&lt;length&gt;</loc> values specified on a <loc href="#layout-vocabulary-region">
+<el>region</el></loc> element using <code>em</code> 
+units are considered relative to the <emph>Computed Cell Size</emph>. When a single value is specified, then it is 
```

Comment:
Is the second sentence of this paragraph "When a single ..." dependent on the context in the first sentence, i.e., does the second sentence only apply for length's using ``em``? Or is a general statement (all lengths)? If the former, then change the second sentence to start with "In that case, when a single ...".